src/main/java/calculator/StringAddCalculator.java

```diff
@@ -0,0 +1,15 @@
+package calculator;
+
+public class StringAddCalculator {
+
+  public static int splitAndSum(String input) {
+    Split split = new Split(input);
+
+    String[] stringArray = split.getStringArray();
+
+    StringSum stringSum = new StringSum(stringArray);
+
+    return stringSum.getSum();
```

Comment:
Split과 StringSum은 input에 대한 output을 내는 유틸리티 클래스 성격이 강해보여요!

이런 경우엔 정적 메소드로 기능을 제공하는 것이 가독성과 사용성 측면에서 더 유리한 것 같아요.

```suggestion
    String[] stringArray = Split.getStringArray(input);

    return StringSum.getSum(stringArray);
```

Reply:
정말 그러네요. 
뭔가 자바를 자바스럽게 아직 사용하는게 많이 미숙한 것 같아, 더 노력해야겠습니다.
말씀하신대로 가독성과 사용성측면에서 훨씬 좋아보이네요.
수정하였습니다!.